templates/persistence/graphdb-master-default-pv.yaml

```diff
@@ -11,11 +11,14 @@
 apiVersion: {{ $.Values.versions.pv }}
 kind: PersistentVolume
 metadata:
-  name: graphdb-master-default-master-{{ $master_index }}-pv
+  name: graphdb-default-master-{{ $master_index }}-pv
```

Comment:
Persistent volumes are not bound in namespaces but are cluster resources. So if you deploy the chart in different namespaces, the PVs will be in a conflict. You can prefix the PV name with {{ $.Release.Namespace }} or {{ $.Release.Name }}